fire-cracker/SendIT-react
# #164974983 Implement User should be able to create order using react

#### What does this PR do?
- Implements create order and header components
- Implements create order and header functionality using actions

#### How should this be manually tested?
- Clone repo
- Check out to branch `feature/164974983/createOrder`
- Run application with `npm run start-dev` in the terminal
- log in and create an order.

#### What are the relevant pivotal tracker stories?
- [Implement create parcel page using react](https://www.pivotaltracker.com/story/show/164974983)
